rsconf/component/rsiviz.py

```diff
@@ -19,27 +19,33 @@ def internal_build_compile(self):
         jc, z = self.j2_ctx_init()
         z._run_u = jc.rsconf_db.run_u
         self.j2_ctx_pksetdefault(
-            PKDict(pykern=PKDict(pkconfig=PKDict(channel=jc.rsconf_db.channel)))
+            PKDict(
+                pykern=PKDict(
+                    pkconfig=PKDict(channel=jc.rsconf_db.channel),
+                    pkasyncio=PKDict(server_port=8002),
+                ),
+                rsiviz=PKDict(pkcli=PKDict(service=PKDict(index_iframe_port=8000))),
+            )
         )
         self.__run_d = systemd.docker_unit_prepare(
             self,
             jc,
+            # TODO(e-carlin): This is the default command (set by build_docker_cmd)
+            # is there a way to just use it and not specify cmd?
             docker_exec=f"bash {db.user_home_path(z._run_u)}/.radia-run/start",
         )
 
     def internal_build_write(self):
-        from rsconf.component import db_bkp
         from rsconf.component import nginx
-        from rsconf.component import docker
 
         jc = self.j2_ctx
         z = jc[self.name]
         d = self.__run_d.join(_DB_SUBDIR)
         jc.nginx.pkupdate(
             index_port=8880,
```

Comment:
This would be good as constants. You could just have a pkdict that is passed to pkupdate.

Reply:
ff62898